cognite/neat/rules/parser.py

```diff
@@ -246,14 +246,14 @@ def from_raw_dataframes(cls, raw_dfs: dict[str, pd.DataFrame]) -> "RawTables":
 
         tables_dict = {
             Tables.metadata: raw_dfs[Tables.metadata],
```

Comment:
Why not on the metadata table?

Reply:
you do not have column names, `metadata` works more as key value pairs, other sheets are actually tables